source/css/scss-common/objects/_collapse.scss

```diff
@@ -235,14 +235,83 @@
   }
 }
 
-.a-third-action-mobile {
-  background-color: $grey-light;
+@media(min-width: 768px) {
+  .addPersonOrBusiness {
+    padding-right: 12px;
+    padding-left: 12px;
+  }
+
+  .rightHolderList {
+    .a-list-container {
+      padding-right: 12px;
+      padding-left: 20px;
+    }
+  }
+}
+
+@media(max-width: 768px) {
```

Comment:
min-width og max-width bør vel ikke være det samme da når størrelsen er 768px vil begge tilfellene være sanne

Reply:
stemmer for å sette `max-width: 767px`

Reply:
stemmer fixed